### ISSUE TITLE
[Crash] Game Crashing when selecting the connecting textures option

### ISSUE THREAD

**Issue by Flaguna2** [SuperMartijn642/Rechiseled#65]
EDIT: Fusion 1.1.0 is the issue. Downgrade to 1.0.6

**Version Info**
- Minecraft, 1.20.1
- Rechiseled, 1.1.5b (latest at the moment)
- Fusion, 1.1.0 (latest at the moment)

Fabric 1.14.24-1.20.1

**Steps to Reproduce**
Simply open the chisel menu, add any block and select the option for connected textures. Crash.
I recall this not happening before the latest update of Fusion, but I might be wrong

I belive all mods are visible in the crash log below, but I can provide a full list of all mods installed, if need be

```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2023-11-10 23:54:22
Description: Rendering screen

java.lang.ClassCastException: class link.infra.indium.renderer.render.AbstractBlockRenderContext$2 cannot be cast to class net.fabricmc.fabric.impl.client.indigo.renderer.mesh.QuadViewImpl (link.infra.indium.renderer.render.AbstractBlockRenderContext$2 and net.fabricmc.fabric.impl.client.indigo.renderer.mesh.QuadViewImpl are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @6ee12bac)
	at net.fabricmc.fabric.impl.client.indigo.renderer.mesh.MutableQuadViewImpl.copyFrom(MutableQuadViewImpl.java:171)
	at com.supermartijn642.fusion.model.types.connecting.ConnectingBakedModel.lambda$emitBlockQuads$0(ConnectingBakedModel.java:84)
	at link.infra.indium.renderer.render.AbstractRenderContext.lambda$new$1(AbstractRenderContext.java:45)
	at link.infra.indium.renderer.render.AbstractRenderContext.transform(AbstractRenderContext.java:63)
	at link.infra.indium.renderer.render.AbstractBlockRenderContext.renderQuad(AbstractBlockRenderContext.java:111)
	at link.infra.indium.renderer.render.AbstractBlockRenderContext$2.emitDirectly(AbstractBlockRenderContext.java:74)
	at link.infra.indium.renderer.mesh.MutableQuadViewImpl.emit(MutableQuadViewImpl.java:261)
	at link.infra.indium.renderer.mesh.MutableQuadViewImpl.emit(MutableQuadViewImpl.java:56)
	at net.fabricmc.fabric.impl.renderer.VanillaModelEncoder.emitBlockQuads(VanillaModelEncoder.java:65)
	at net.minecraft.class_1087.emitBlockQuads(class_1087.java:1039)
	at com.supermartijn642.fusion.model.types.connecting.ConnectingBakedModel.emitBlockQuads(ConnectingBakedModel.java:94)
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.emitBlockQuads(ForwardingBakedModel.java:51)
	at me.pepperbell.continuity.client.model.EmissiveBakedModel.emitBlockQuads(EmissiveBakedModel.java:64)
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.emitBlockQuads(ForwardingBakedModel.java:51)
	at me.pepperbell.continuity.client.model.EmissiveBakedModel.emitBlockQuads(EmissiveBakedModel.java:72)
	at link.infra.indium.renderer.render.NonTerrainBlockRenderContext.render(NonTerrainBlockRenderContext.java:66)
	at net.minecraft.class_778.handler$cll000$indium$hookRender(class_778.java:553)
	at net.minecraft.class_778.method_3374(class_778.java)
	at com.supermartijn642.rechiseled.screen.ScreenBlockRenderer.renderBlock(ScreenBlockRenderer.java:68)
	at com.supermartijn642.rechiseled.screen.ScreenBlockRenderer.drawBlock(ScreenBlockRenderer.java:50)
	at com.supermartijn642.rechiseled.screen.EntryPreviewWidget.render(EntryPreviewWidget.java:80)
	at com.supermartijn642.core.gui.widget.BaseWidget.lambda$render$3(BaseWidget.java:140)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at com.supermartijn642.core.gui.widget.BaseWidget.render(BaseWidget.java:140)
	at com.supermartijn642.core.gui.WidgetContainerScreen.method_25394(WidgetContainerScreen.java:113)
	at net.minecraft.class_437.method_47413(class_437.java:110)
	at net.minecraft.class_757.method_3192(class_757.java:945)
	at net.minecraft.class_310.method_1523(class_310.java:1219)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.fabric.impl.client.indigo.renderer.mesh.MutableQuadViewImpl.copyFrom(MutableQuadViewImpl.java:171)
	at com.supermartijn642.fusion.model.types.connecting.ConnectingBakedModel.lambda$emitBlockQuads$0(ConnectingBakedModel.java:84)
	at link.infra.indium.renderer.render.AbstractRenderContext.lambda$new$1(AbstractRenderContext.java:45)
	at link.infra.indium.renderer.render.AbstractRenderContext.transform(AbstractRenderContext.java:63)
	at link.infra.indium.renderer.render.AbstractBlockRenderContext.renderQuad(AbstractBlockRenderContext.java:111)
	at link.infra.indium.renderer.render.AbstractBlockRenderContext$2.emitDirectly(AbstractBlockRenderContext.java:74)
	at link.infra.indium.renderer.mesh.MutableQuadViewImpl.emit(MutableQuadViewImpl.java:261)
	at link.infra.indium.renderer.mesh.MutableQuadViewImpl.emit(MutableQuadViewImpl.java:56)
	at net.fabricmc.fabric.impl.renderer.VanillaModelEncoder.emitBlockQuads(VanillaModelEncoder.java:65)
	at net.minecraft.class_1087.emitBlockQuads(class_1087.java:1039)
	at com.supermartijn642.fusion.model.types.connecting.ConnectingBakedModel.emitBlockQuads(ConnectingBakedModel.java:94)
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.emitBlockQuads(ForwardingBakedModel.java:51)
	at me.pepperbell.continuity.client.model.EmissiveBakedModel.emitBlockQuads(EmissiveBakedModel.java:64)
	at net.fabricmc.fabric.api.renderer.v1.model.ForwardingBakedModel.emitBlockQuads(ForwardingBakedModel.java:51)
	at me.pepperbell.continuity.client.model.EmissiveBakedModel.emitBlockQuads(EmissiveBakedModel.java:72)
	at link.infra.indium.renderer.render.NonTerrainBlockRenderContext.render(NonTerrainBlockRenderContext.java:66)
	at net.minecraft.class_778.handler$cll000$indium$hookRender(class_778.java:553)
	at net.minecraft.class_778.method_3374(class_778.java)
	at com.supermartijn642.rechiseled.screen.ScreenBlockRenderer.renderBlock(ScreenBlockRenderer.java:68)
	at com.supermartijn642.rechiseled.screen.ScreenBlockRenderer.drawBlock(ScreenBlockRenderer.java:50)
	at com.supermartijn642.rechiseled.screen.EntryPreviewWidget.render(EntryPreviewWidget.java:80)
	at com.supermartijn642.core.gui.widget.BaseWidget.lambda$render$3(BaseWidget.java:140)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at com.supermartijn642.core.gui.widget.BaseWidget.render(BaseWidget.java:140)
	at com.supermartijn642.core.gui.WidgetContainerScreen.method_25394(WidgetContainerScreen.java:113)
	at net.minecraft.class_437.method_47413(class_437.java:110)

-- Screen render details --
Details:
	Screen name: com.supermartijn642.core.gui.WidgetContainerScreen
	Mouse location: Scaled: (360, 225). Absolute: (720.000000, 450.000000)
	Screen size: Scaled: (720, 450). Absolute: (1440, 900). Scale factor of 2.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:945)
	at net.minecraft.class_310.method_1523(class_310.java:1219)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Affected level --
Details:
	All players: 1 total; [class_746['Flaguna'/203, l='ClientLevel', x=98.11, y=100.00, z=-87.27]]
	Chunk stats: 4096, 1529
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,111,0), Section: (at 0,15,0 in 0,6,0; chunk contains blocks 0,-64,0 to 15,319,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 58921 game time, 58921 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:458)
	at net.minecraft.class_310.method_1587(class_310.java:2406)
	at net.minecraft.class_310.method_1514(class_310.java:821)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, file/§dOverlayed§5.zip, file/Spawn+Eggs+Faces+1.19.zip, fabric, Moonlight Mods Dynamic Assets, visualoverhaul:rounddiscs, diagonalblocks:default_block_models

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 4497643816 bytes (4289 MiB) / 9965666304 bytes (9504 MiB) up to 10468982784 bytes (9984 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 2600 Six-Core Processor            
	Identifier: AuthenticAMD Family 23 Model 8 Stepping 2
	Microarchitecture: Zen+
	Frequency (GHz): 3.39
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce GTX 1050 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1c82
	Graphics card #0 versionInfo: DriverVersion=31.0.15.3623
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 32681.12
	Virtual memory used (MB): 17650.54
	Swap memory total (MB): 16384.00
	Swap memory used (MB): 161.47
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx9983M -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		accurateblockplacement: Accurate Block Placement 1.0.17
		additionaladditions: Additional Additions 6.0.1
			libgui: LibGui 8.0.1+1.20
				jankson: Jankson 5.0.1+j1.2.2
					blue_endless_jankson: jankson 1.2.2
				libninepatch: LibNinePatch 1.2.0
		additionalstructures: Additional Structures 4.2.0
		adorn: Adorn 5.0.0+1.20.1
		ambientsounds: AmbientSounds 5.2.24
		architectury: Architectury 9.1.12
		athena: Athena 3.0.0
		betteranimationscollection: Better Animations Collection 8.0.0
		betterarcheology: Better Archeology 1.0.2
		betterclouds: Better Clouds 1.3.8+1.20.1
		bettercombat: Better Combat 1.8.0+1.20.1
		bettercommandblockui: BetterCommandBlockUI 0.4.2
		betterdeserttemples: YUNG's Better Desert Temples 1.20-Fabric-3.0.3
			org_reflections_reflections: reflections 0.10.2
		betterdungeons: YUNG's Better Dungeons 1.20-Fabric-4.0.3
		betterendisland: YUNG's Better End Island 1.20-Fabric-2.0.4
		betterfortresses: YUNG's Better Nether Fortresses 1.20-Fabric-2.0.5
		betterjungletemples: YUNG's Better Jungle Temples 1.20-Fabric-2.0.4
		bettermineshafts: YUNG's Better Mineshafts 1.20-Fabric-4.0.4
		betteroceanmonuments: YUNG's Better Ocean Monuments 1.20-Fabric-3.0.3
		betterstrongholds: YUNG's Better Strongholds 1.20-Fabric-4.0.3
		betterwitchhuts: YUNG's Better Witch Huts 1.20-Fabric-3.0.3
		blasting_plus_mr: Blasting Plus 1.1
		blockus: Blockus 2.7.6+1.20.1
			terraform-wood-api-v1: Terraform Wood API (v1) 7.0.1
		brrp_v1: Better Runtime Resource Pack (BRRP) v1 1.0.0
		carpeted: Carpeted 1.20-1.4
		carrier: Carrier 1.12.0
			advanced_runtime_resource_pack: Runtime Resource Pack 0.6.7
		chipped: Chipped 3.0.1
		cloth-config: Cloth Config v11 11.1.106
			cloth-basic-math: cloth-basic-math 0.6.1
		clumps: Clumps 12.0.0.3
		clutter: Clutter 1.20.1-0.3.7
		collective: Collective 7.3
		connectiblechains: Connectible Chains 2.2.1+1.20.1
		continuity: Continuity 3.0.0-beta.3+1.20.1
		corgilib: CorgiLib 4.0.0.0
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
		crawl: Crawl 0.12.0
			mm: Manningham Mills 2.3
		creativecore: CreativeCore 2.11.9
			net_minecraftforge_eventbus: eventbus 6.0.3
		dawn: Dawn API 5.0.0
			terraform-shapes-api-v1: Terraform Shapes API (v1) 7.0.1
		diagonalfences: Diagonal Fences 8.1.2
			diagonalblocks: Diagonal Blocks 8.0.2
		disenchanter: Disenchanter 0.1.7+1.20
		effortless: Effortless Fabric 1.6.4
		enhancedcelestials: Enhanced Celestials 5.0.0.2
		enhancedvisuals: EnhancedVisuals 1.6.9
		entity_model_features: Entity Model Features 1.1.0
		entity_texture_features: Entity Texture Features 4.6.1
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		entityculling: EntityCulling-Fabric 1.6.2-mc1.20
		everycomp: Every Compat 1.20-2.6.16
		explorify: Explorify v1.3.0-mc1.20u1.20.1
		explosiveenhancement: Explosive Enhancement 1.2.2-1.20.x
		extshape: Extended Block Shapes 2.0.3
		extshape_blockus: Extended Block Shapes - Blockus 2.0.3
		fabric-api: Fabric API 0.90.7+1.20.1
			fabric-api-base: Fabric API Base 0.4.30+7abfd51577
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.35+4d8536c977
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.12+215bbe9677
			fabric-block-api-v1: Fabric Block API (v1) 1.0.10+92a0d36777
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.0+92a0d36777
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.40+b3afc78b77
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.1+97bb207577
			fabric-command-api-v1: Fabric Command API (v1) 1.2.33+f71b366f77
			fabric-command-api-v2: Fabric Command API (v2) 2.2.12+b3afc78b77
			fabric-commands-v0: Fabric Commands (v0) 0.2.50+df3654b377
			fabric-containers-v0: Fabric Containers (v0) 0.1.63+df3654b377
			fabric-content-registries-v0: Fabric Content Registries (v0) 4.0.10+57aed33f77
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.4+a1a980da77
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.18+aeb40ebe77
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 12.3.2+d7b148e077
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.53+8536527b77
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.22+b3afc78b77
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.1+e91849a877
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.62+df3654b377
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.39+ae9f657a77
			fabric-item-api-v1: Fabric Item API (v1) 2.1.27+b3afc78b77
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.11+d7b148e077
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.36+fb8d95da77
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.34+df3654b377
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.21+b3afc78b77
			fabric-loot-api-v2: Fabric Loot API (v2) 1.2.0+96dfa95977
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.44+9e7660c677
			fabric-message-api-v1: Fabric Message API (v1) 5.1.8+d7b148e077
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.49+b3afc78b77
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 1.0.2+709a987177
			fabric-models-v0: Fabric Models (v0) 0.4.1+9386d8a777
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.10+eeb8eb3677
			fabric-networking-v0: Fabric Networking (v0) 0.3.50+df3654b377
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 11.1.2+4ee0bc6077
			fabric-particles-v1: Fabric Particles (v1) 1.1.1+201a23a077
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.20+b3afc78b77
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.3.2+4df89eb277
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.2.0+39a511ba77
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.5.0+39a511ba77
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.45+df3654b377
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.36+92a0d36777
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.27+b3afc78b77
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.48+df3654b377
			fabric-rendering-v1: Fabric Rendering (v1) 3.0.7+b3afc78b77
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.6+d7b148e077
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.9+132c48c177
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.7+b3afc78b77
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.29+b3afc78b77
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.12+b3afc78b77
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.3.2+1bc31aeb77
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 4.3.0+6c31357e77
		fabric-language-kotlin: Fabric Language Kotlin 1.10.13+kotlin.1.9.20
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.9.20
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.9.20
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.9.20
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.9.20
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.22.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.7.3
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.7.3
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.1
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.0
		fabricloader: Fabric Loader 0.14.24
		fallingtree: FallingTree 4.1.1
		forgeconfigapiport: Forge Config API Port 8.0.0
		friendsandfoes: Friends&Foes 1.9.8
		fusion: Fusion 1.1.0
		geckolib: GeckoLib 4 4.2.4
			com_eliotlash_mclib_mclib: mclib 20
		graveyard: The Graveyard 2.9.1
			omega-config: OmegaConfig 1.2.2-1.18.1
		handcrafted: Handcrafted 3.0.5
		immersive_aircraft: Immersive Aircraft 0.6.2+1.20.1
		imst: Immersive structures 2.1.0
		indium: Indium 1.0.27+mc1.20.1
		inventorysorter: Inventory Sorter 1.9.0-1.20
			kyrptconfig: Kyrpt Config 1.5.6-1.20
		java: OpenJDK 64-Bit Server VM 17
		justhammers: Just Hammers 2.0.0+mc1.20.1
		lambdynlights: LambDynamicLights 2.3.2+1.20.1
			pride: Pride Lib 1.2.0+1.19.4
			spruceui: SpruceUI 5.0.0+1.20
		libertyvillagers: Liberty's Villagers 1.0.14+fabric+1.20.1
			server_translations_api: Server Translations API 2.0.0+1.20
				packet_tweaker: Packet Tweaker 0.4.0+1.19.4
		linkart: Linkart 5.1.1-1.19.3
		lithium: Lithium 0.11.2
		lunaslimes: Luna Slimes 1.6.1-1.20
		memoryleakfix: Memory Leak Fix 1.1.2
		midnightlib: MidnightLib 1.4.1
		minecraft: Minecraft 1.20.1
		modmenu: Mod Menu 7.2.2
		moonlight: Moonlight 1.20-2.8.59
		mr_animated_doors: Animated Doors 1.0
		particletweaks: Particle Tweaks 1.1.3-Fabric+1.20.1-unstable
		player-animator: Player Animator 1.0.2-rc1+1.20
		polymorph: Polymorph 0.49.1+1.20.1
			cardinal-components-item: Cardinal Components API (items) 5.2.1
			spectrelib: SpectreLib 0.13.14+1.20.1
		presencefootsteps: Presence Footsteps 1.9.0
			kirin: Kirin UI 1.15.0
		puzzleslib: Puzzles Lib 8.1.8
			puzzlesaccessapi: Puzzles Access Api 8.0.7
		quartzelv: Quartz Elevator 2.2.5+1.20
		rechiseled: Rechiseled 1.1.5+b
		repurposed_structures: Repurposed Structures 7.1.11+1.20.1-fabric
		resourcefullib: Resourceful Lib 2.1.16
			com_teamresourceful_bytecodecs: bytecodecs 1.0.2
			com_teamresourceful_yabn: yabn 1.0.3
		roughlyenoughitems: Roughly Enough Items 12.0.652
			error_notifier: Error Notifier 1.0.9
		shouldersurfing: Shoulder Surfing Reloaded 2.6
		simple_copper_pipes: Simple Copper Pipes 1.14
		simplyswords: Simply Swords 1.48.1-1.21
		skinlayers: 3d Skin Layers 1.5.3-mc1.20
		smoothparticles: Smooth Particles 1.2.0-1.21
		smoothswapping: Smooth Swapping 0.9.3.1
		sodium: Sodium 0.5.3
		starlight: Starlight 1.1.2+fabric.dbc156f
		superflatworldnoslimes: Superflat World No Slimes 3.1
		supermartijn642configlib: SuperMartijn642's Config Lib 1.1.8+a
		supermartijn642corelib: SuperMartijn642's Core Lib 1.1.15
		supplementaries: Supplementaries 1.20-2.6.19
			mixinsquared: MixinSquared 0.1.1
		taxfreelevels: Tax Free Levels 1.3.3
		universal_ores: Universal Ores 1.5.2
		unloadedactivity: Unloaded Activity 0.4.4
			cardinal-components-base: Cardinal Components API (base) 5.2.2
			cardinal-components-block: Cardinal Components API (blocks) 5.2.2
			cardinal-components-chunk: Cardinal Components API (chunks) 5.2.2
			cardinal-components-entity: Cardinal Components API (entities) 5.2.2
		visuality: Visuality 0.7.0+1.20
		visualoverhaul: VisualOverhaul 5.0.1
		wilderwild: Wilder Wild 2.1.12-Fabric+1.20.1
			frozenlib: FrozenLib 1.4.1-Fabric+1.20.1
				com_github_llamalad7_mixinextras: MixinExtras 0.2.1-beta.2
		windchimes: Windchimes 1.2.3+1.20
		yet_another_config_lib_v3: YetAnotherConfigLib 3.0.3+1.20
			com_twelvemonkeys_common_common-image: common-image 3.9.4
			com_twelvemonkeys_common_common-io: common-io 3.9.4
			com_twelvemonkeys_common_common-lang: common-lang 3.9.4
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.9.4
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.9.4
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.9.4
		yungsapi: YUNG's API 1.20-Fabric-4.0.2
			org_javassist_javassist: javassist 3.29.2-GA
		yungsbridges: YUNG's Bridges 1.20-Fabric-4.0.3
		yungsextras: YUNG's Extras 1.20-Fabric-4.0.3
	Launched Version: fabric-loader-0.14.24-1.20.1
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce GTX 1050 Ti/PCIe/SSE2 GL version 3.2.0 NVIDIA 536.23, NVIDIA Corporation
	Window size: 1440x900
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, file/§dOverlayed§5.zip (incompatible), file/Spawn+Eggs+Faces+1.19.zip (incompatible), fabric
	Current Language: en_us
	CPU: 12x AMD Ryzen 5 2600 Six-Core Processor 
	Server Running: true
	Player Count: 1 / 8; [class_3222['Flaguna'/203, l='ServerLevel[Tranças do rei careca]', x=98.11, y=100.00, z=-87.27]]
	Data Packs: vanilla, fabric, Everycomp Generated Pack, Moonlight Mods Dynamic Assets, Supplementaries Generated Pack, bundle
	Enabled Feature Flags: minecraft:bundle, minecraft:vanilla
	World Generation: Stable
```

**Comment by aqua-air** [comment]
Let me know if you find a fix. I think i'm having the same issue. I just crash when i try to break any rechiseled blocks. crash report points to fusion. I know you can bypass the crash by breaking the block in creative mode or using block replacer item

**Comment by claw1200** [comment]
I guess the solution right now is to disable Fusion until this gets patched, obviously you wont have connected textures but at least you wont crash if you accidently click the button in Rechisel.

**Comment by SuperMartijn642** [comment]
For now, you can just downgrade Fusion to 1.0.6. Rechiseled doesn't use any features added in Fusion 1.1.0.

**Comment by Flaguna2** [comment]
> For now, you can just downgrade Fusion to 1.0.6. Rechiseled doesn't use any features added in Fusion 1.1.0.

yup, this is the issue. Ty!

**Comment by SuperMartijn642** [comment]
Fixed now in Fusion version 1.1.0a.
Thank you for reporting the issue!
